controllers/imagerepository_controller.go

```diff
@@ -77,6 +77,22 @@ func (r *ImageRepositoryReconciler) Reconcile(req ctrl.Request) (ctrl.Result, er
 
 	log := r.Log.WithValues("controller", strings.ToLower(imagev1alpha1.ImageRepositoryKind), "request", req.NamespacedName)
 
+	if imageRepo.Spec.Suspend {
+		msg := "ImageRepository is suspended, skipping reconciliation"
```

Comment:
Nit pick on the language: reconciliation is completed, not skipped; the thing being skipped is scanning of the image repository.

Reply:
I retrained the same wording as https://github.com/fluxcd/kustomize-controller/blob/master/controllers/kustomization_controller.go#L110 to avoid confusion and this is definitely arguable, it's not really reconciled (as folks would understand it) because the reconcile loop terminates early.

Reply:
> I retained the same wording as [the kustomize controller]

Hmm, yeah, I think it's wrong there -- I have taken it up [in a discussion](https://github.com/fluxcd/toolkit/discussions/164). Let's stick with consistency here for now, as you prefer.